clang/lib/Sema/SPIRVBuiltins.td

```diff
@@ -926,9 +926,9 @@ foreach name = ["GroupAll", "GroupAny"] in {
 
 foreach name = ["GroupBroadcast"] in {
   foreach IDType = TLAllInts.List in {
-    def : SPVBuiltin<name, [AGenType1, UInt, AGenType1, IDType], Attr.Convergent>;
-    def : SPVBuiltin<name, [AGenType1, UInt, AGenType1, VectorType<IDType, 2>], Attr.Convergent>;
-    def : SPVBuiltin<name, [AGenType1, UInt, AGenType1, VectorType<IDType, 3>], Attr.Convergent>;
+    def : SPVBuiltin<name, [AGenTypeN, UInt, AGenTypeN, IDType], Attr.Convergent>;
+    def : SPVBuiltin<name, [AGenTypeN, UInt, AGenTypeN, VectorType<IDType, 2>], Attr.Convergent>;
+    def : SPVBuiltin<name, [AGenTypeN, UInt, AGenTypeN, VectorType<IDType, 3>], Attr.Convergent>;
```

Comment:
I have a limited understanding of the changes here, and would appreciate if the runtime reviewers would also review this cfe part.   Thanks.

Reply:
This change looks fine to me. Essentially we say that those `GroupBroadcast` overloads accept not only scalars, but vectors as well as defined by `AGenTypeN` alias in the beginning of this file